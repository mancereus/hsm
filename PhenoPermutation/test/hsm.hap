3 6
1 0 0 0
11111
11111
2 0 0 0
21211
22121
3 0 0 0
11112
12221
