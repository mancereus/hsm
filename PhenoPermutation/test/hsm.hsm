0 0.4444444444444436 0.8540872078149434 0.2160313235872755 0.39 0.39 
1 0.4444444444444432 0.8540872078149432 0.21603132358727561 0.39 0.39 
2 0.4444444444444432 0.8540872078149432 0.21603132358727561 0.39 0.39 
3 -0.22222222222222288 0.6216392291597141 0.28071092797543107 0.39 0.39 
4 1.7777777777777772 1.8832504146693634 0.05919075357765635 0.0 0.0 
