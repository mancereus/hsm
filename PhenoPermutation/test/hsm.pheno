id sex status
id1 0 0
id2 0 1
id3 0 1

