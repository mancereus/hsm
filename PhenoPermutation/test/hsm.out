0 0.4444444444444436 -99.0 -99.0 
1 0.4444444444444432 -99.0 -99.0 
2 0.4444444444444432 -99.0 -99.0 
3 -0.22222222222222288 -99.0 -99.0 
4 1.7777777777777772 -99.0 -99.0 
